### PR TITLE
Tweaks

### PR DESCRIPTION
**Pull request by bratseth**
Consider these ...

### CODE REVIEW

File: vagrant/Vagrantfile [hunk 1]
```diff
@@ -4,7 +4,8 @@
 # For a complete reference, please see the online documentation at https://docs.vagrantup.com.
 Vagrant.configure("2") do |config|
 
-  config.vm.box = "boxcutter/centos7-desktop"
+  config.vm.box = "centos7-desktop"
+  config.vm.box_url = "/Users/bratseth/development/github/vespa-engine/vespa/vagrant/centos7-desktop-17.0915.1.box"
```
Comment (vekterli) [inline vagrant/Vagrantfile]: I think this will not work for most users..! 😉

Comment (bratseth) [reply vagrant/Vagrantfile]: Thanks. Since we don't have a working solution atm I just put TODO.

File: vagrant/Vagrantfile [hunk 2]
```diff
@@ -25,6 +26,8 @@ Vagrant.configure("2") do |config|
     yum -y install epel-release
     yum -y install centos-release-scl
     yum -y install yum-utils
+    yum -y install boost-devel
```
Comment (geirst) [inline vagrant/Vagrantfile]: These two should be removed. They are installed by "yum-builddep -y /vagrant/dist/vespa.spec" further down.

**Comment by geirst** [comment]
@bratseth please look at my comment

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**